procedures/meeting-observers.md

```diff
@@ -0,0 +1,39 @@
+# Meeting observers
+
+## Motivation
+
+There has been some interest in people joining meetings in a pure observational manner. (See [previous discussion on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Observing.20meetings.3F).) Further discussion on the specifics of a proposal happened on [Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Meeting.20observers) and [in a council meeting](https://github.com/rust-lang/leadership-council/blob/245b22a105546a680aad582cf856311a80e93baf/minutes/sync-meeting/2023-07-20.md).
+
```

Comment:
I think it'd be good to add some goals here. Right now it's sort of like "there was interest (from who?) so we decided to do it." I might suggest something like:

```suggestion
### Goals

* Increase transparency and trust between the Council and the rest of the Rust project.
* Give the Council the chance to model decision making and processes for resolving disagreements.
* Retain the ability to efficiently and effectively use scarce synchronous meeting time.
* Avoid creating opportunities for harassment, which could particularly make it hard for members of marginalized groups to serve as representatives on the council.
```

Reply:
I've moved the considerations section back to the top, which I think covers this.